packages/common/src/helpers/parsers.ts

```diff
@@ -32,6 +32,14 @@ export const isBoolean = (data: unknown): data is boolean =>
 export const isString = (data: unknown): data is string =>
   typeof data === 'string'
 
+export const isKeyOf = <T extends Record<string, unknown>>(
```

Comment:
if it was exported, probably it didn't build while you were coding above, that's why imported with the path